### PR TITLE
Adding 2 new streams use case logo

### PR DESCRIPTION
**Pull request by manjuapu**
Michael, Nick,
Can you please review this?
Thanks!!
@guozhangwang I need help with merging to Kafka site.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1341,7 +1341,7 @@ ul.video-list {
     overflow: hidden;
 }
 .extra__space{
-    padding-bottom:2.8rem;
+    padding-bottom:1rem;
```
Comment (ewencp) [inline css/styles.css:1344]: Not sure if this is the cause, but it looks like the bottom of the left column's HomeAway item is "overflowing" into the next column or something.

See the top

screen shot 2018-02-27 at 7 37 37 pm

and bottom

screen shot 2018-02-27 at 7 37 42 pm

as they show up in my browser.

**Review by ewencp — COMMENTED** [review]
@manjuapu Have one formatting/styling detail that needs to be sorted out.

Haven't finished the release, but as some stuff is staged, we'll also need to port for `11/streams/index.html`.

**Comment by ewencp** [comment]
Also, @manjuapu this part of the docs is maintained upstream in the main Kafka repo since it is versioned for different releases (with latest version linked to the default, current docs). I think we'd want this upstream in the main Kafka repo also, otherwise we'll just lose this with the next release.

**Comment by manjuapu** [comment]
@ewencp  I usually add it to upstream also, when I make edits to Kafka-site. https://github.com/apache/kafka/blob/trunk/docs/streams/index.html

For styling issue, please clear cache in your browser, I am not sure how to clear cache on apache Kafka site.

**Comment by manjuapu** [comment]
@ewencp I have added Funding circle and Homeaway in this PR,  I think it is on hold, more info here- https://confluentinc.atlassian.net/browse/WEBR-194?focusedCommentId=44691&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-44691

**Comment by mjsax** [comment]
Closing this PR as abandoned. (We can reopen if necessary.)

**Pull request closed without merging**